src/payment/checkout/payment-form/PaymentForm.jsx

```diff
@@ -198,13 +203,30 @@ export class PaymentFormComponent extends React.Component {
       isQuantityUpdating,
     } = this.props;
 
+    // Stripe element config
+    // TODO: Move these to a better home
+    const appearance = {
+      theme: 'stripe',
+    };
+    const options = {
+      clientSecret: this.props.captureKeyId,
+      appearance,
+    };
+    const tempFlag = false;
     let submitButtonState = 'default';
     // istanbul ignore if
     if (disabled) { submitButtonState = 'disabled'; }
     // istanbul ignore if
     if (isProcessing) { submitButtonState = 'processing'; }
     return (
       <ErrorFocusContext.Provider value={this.state.firstErrorId}>
+        {options.clientSecret && (
```

Comment:
Flag will be that stripe should be used, so I believe you can use a ternary operator here with L229?